### ISSUE TITLE
DAS diagnosics completion page doesn't include html completions (via plugin)

### ISSUE THREAD

**Issue by MichaelRFairhurst** [dart-lang/sdk#34562]
Looks like it only tracks completions to dart files, but for dart files it does seem to include plugin results.

**Comment by bwilkerson** [comment]
Correct. The status pages do not currently have any way to get information from plugins. The completion performance is maintained by `CompletionDomainHandler`, but that only includes the completions generated in server. Completions generated in plugins don't have any way to report their performance back to server.

The plugin spec has a TODO to provide notifications by which plugins can report instrumentation and/or DartSilo data. Getting performance data onto the completion page would require a similar notification by which performance data could be reported back to server.
